notebooks/219-knowledge-graphs-conve/README.md

```diff
@@ -5,7 +5,7 @@
 Commonly, knowledge graphs are used to encode domain information by representing them as factual triples of source, relation and target entities.
 For ex. For the countries domain, the domain knowledge can be encoded by representing the countries as entities (for ex. Germany, Poland) and relation between them (Neighbor) as factual triples (Poland, Neighbor, Germany) as well as other information as follows:
 <p style="text-align:center;">
-    <img src="./images/countries_knowledge_graph.png" width=500>
+    <img src="../data/image/countries_knowledge_graph.png" width=500>
```

Comment:
An URL of the image uploaded to Github should replace this like it's done in other README files. The countries_knowledge_graph.png file should be removed.